recipes-graphics/userland/userland.inc

```diff
@@ -0,0 +1,38 @@
+DESCRIPTION = "This repository contains the source code for the ARM side \
```

Comment:
Can't we just have a separate package inside the userland recipe for the EEPROM tools? Wouldn't userland compile?

Reply:
we don't include userland when choosing vc4graphics, so a separate package for these tools is a fine approach.

Reply:
Good point.